File: JXBC-Server/BossComments/apiserver/appbase/models/AccountSign.php
<?php

namespace app\appbase\models;

/**
 * @SWG\Definition(required={"MobilePhone","Password"})
 */
class AccountSign
{   
    /**
     * @SWG\Property(type="string")
     */
    public $MobilePhone;

    /**
     * @SWG\Property(type="integer")
     */
    public $Password;
    
    /**
     * @SWG\Property(type="string", description="短信验证码")
     */
    public $ValidationCode;
    
    /**
     * @SWG\Property(type="integer", description="注册时选择的身份")
     */
    public $SelectedProfileType;    
    
    /**
     * @SWG\Property(type="string", description="注册邀请码")
     */
    public $InviteCode;    
}

File: jxbc-android/BossComment/bosscomments/src/main/java/com/juxian/bosscomments/models/AccountSign.java
package com.juxian.bosscomments.models;

public class AccountSign extends BaseEntity {

	public static final int USER_PROFILE = 1;
	public static final int ORGANIZATION_PROFILE = 2;

	public String MobilePhone;
	public String Password;
	public String ValidationCode;
	public String RealName;
	public AdditionalAction AdditionalAction;
	public String Nickname;
	public String CurrentCompany;
	public String CurrentIndustry;
	public String CurrentJobCategory;
	public int SelectedProfileType;
	public String InviteCode;
}